client/my-sites/stats/features/modules/stats-devices/stats-module-devices-wrapper.tsx

```diff
@@ -22,19 +22,17 @@ const StatsModuleDevicesWrapper: React.FC< StatsAdvancedModuleWrapperProps > = (
 	className,
 } ) => {
 	const isNewEmptyStateEnabled = config.isEnabled( 'stats/empty-module-traffic' );
-	const isGatedByShouldGateStats = config.isEnabled( 'stats/restricted-dashboard' );
 	const { devices: devicesStrings } = statsStrings();
 	const shouldGateStats = useShouldGateStats( STATS_TYPE_DEVICE_STATS );
 
 	// Check if blog is internal.
 	const { isPending: isFetchingUsage, data: usageData } = usePlanUsageQuery( siteId );
-	const { isLoading: isLoadingFeatureCheck, supportCommercialUse } = useStatsPurchases( siteId );
+	const { isLoading: isLoadingFeatureCheck } = useStatsPurchases( siteId );
 
 	const isSiteInternal = ! isFetchingUsage && usageData?.is_internal;
 	const isFetching = isFetchingUsage || isLoadingFeatureCheck; // This is not fetching Devices data.
 
-	const isAdvancedFeatureEnabled =
-		isSiteInternal || ( isGatedByShouldGateStats ? ! shouldGateStats : supportCommercialUse );
+	const isAdvancedFeatureEnabled = isSiteInternal || ! shouldGateStats;
```

Comment:
Won't this expose the advanced features to everyone?

Reply:
No, the Devices module is gated with [shouldGateStats](https://github.com/Automattic/wp-calypso/pull/93384/files/c2af4de6788c32f007e38b189088b073ca54f2d6#diff-0e11044d19ddadbb5f285305ca03b520e15442312488397144df4a2cf7e61860R26) here.